nightfall-optimist/src/routes/proposer.mjs

```diff
@@ -134,7 +156,13 @@ router.post('/register', auth, async (req, res, next) => {
         }
       }
     }
-    res.json({ receipt });
+
+    if (signedTx) {
```

Comment:
This `if` is not needed. If `signedTx` is `null`, then something wrong happened, right?

Reply:
Addressed in these commits [d8c542f](https://github.com/EYBlockchain/nightfall_3/pull/1262/commits/d8c542f8a22ab5163b9921975cfb2eb1203646f5), [5fb74e3](https://github.com/EYBlockchain/nightfall_3/pull/1262/commits/5fb74e347080f688f49bb85abe3e9f9c953d3d9e) (I cleaned up all nonce stuff as discussed)